data/layout/telescope_positions-North-1LST.ecsv

```diff
@@ -1,17 +1,12 @@
-# %ECSV 0.9
+# %ECSV 1.0
 # ---
 # datatype:
 # - {name: telescope_name, datatype: string}
 # - {name: pos_x, unit: m, datatype: float64}
 # - {name: pos_y, unit: m, datatype: float64}
 # - {name: pos_z, unit: m, datatype: float64}
-# - {name: lat, unit: deg, datatype: float64}
-# - {name: lon, unit: deg, datatype: float64}
-# - {name: utm_east, unit: m, datatype: float64}
-# - {name: utm_north, unit: m, datatype: float64}
-# - {name: alt, unit: m, datatype: float64}
 # meta:
-#   EPSG: 32628
+#   EPSG: 32719
```

Comment:
This is wrong, the North site is in the 32628 zone (see https://epsg.io/32628).

Reply:
Thanks for pointing this error out. I guess I updated all the headers without looking at the site. I corrected it now.